### PR TITLE
Admin Module-List Food

### PR DESCRIPTION
**Pull request by vhhieu95**

### CODE REVIEW

File: resources/views/foods/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+@section('main-content')
+    <div class="box">
+        <div class="box-header">
+            <h3 class="box-title">{{ trans('foods/listfoods.title') }}</h3>
+            <a href="{{ route('foods.create') }}" class="btn btn-primary btn-xl pull-right fa fa-plus"> {{ trans('foods/listfoods.add-food') }}</a>
+        </div>
+        <!-- /.box-header -->
+        <div class="box-body">
+            <div id="example1_wrapper" class="dataTables_wrapper form-inline dt-bootstrap">
```
Comment (ghost) [inline resources/views/foods/index.blade.php]: Change div id ```example1_wrapper```. Pls remember do not used id with name ```example``` or like that.

File: resources/views/foods/index.blade.php [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                <div class="row">
+                    <div class="col-sm-12">
+                        <table id="example1" class="table table-bordered table-striped dataTable table-hover"
+                               role="grid"
+                               aria-describedby="example1_info">
```
Comment (ghost) [inline resources/views/foods/index.blade.php]: Change ```example1_info ```. Pls remember do not used  ```example``` or like that.

File: resources/views/foods/index.blade.php [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                            <thead>
+                            <tr role="row">
+                                <th style="width: 1em">{{ trans('foods/listfoods.id') }}</th>
```
Comment (ghost) [inline resources/views/foods/index.blade.php]: Do not used css inline

File: resources/views/layouts/partials/script.blade.php [hunk 4]
```diff
@@ -14,3 +14,4 @@
 <script src="{{ asset("/bower_components/AdminLTE/plugins/datatables/jquery.dataTables.min.js") }}"></script>
 
 <script src="{{  asset("/js/dataTable.js") }}"></script>
+
```
Comment (ghost) [inline resources/views/layouts/partials/script.blade.php]: Remove line null

File: resources/views/layouts/partials/style.blade.php [hunk 5]
```diff
@@ -6,4 +6,5 @@
 <link rel="stylesheet" href="{{ asset("/bower_components/AdminLTE/dist/css/skins/skin-blue.min.css") }}">
 <link rel="stylesheet" href="{{ asset("/bower_components/AdminLTE/plugins/datatables/dataTables.bootstrap.css") }}">
 <link href="{{ asset('css/app.css') }}" rel="stylesheet">
-<link href="{{ asset('css/dataTable.css') }}" rel="stylesheet">
\ No newline at end of file
+<link href="{{ asset('css/dataTable.css') }}" rel="stylesheet">
+
```
Comment (ghost) [inline resources/views/layouts/partials/style.blade.php]: Remove line null

File: app/Food.php [hunk 6]
```diff
@@ -18,4 +20,14 @@ public function orderItems()
     {
         return $this->morphMany(OrderItem::class, 'itemtable');
     }
+
+    /**
+     * Food has one Category
+     *
+     * @return Illuminate\Database\Eloquent\Relations\belongsTo
+     */
+    public function categories()
```
Comment (ghost) [inline app/Food.php]: -you should change categories to category. because food has only a category.

File: app/Http/Controllers/FoodController.php [hunk 7]
```diff
@@ -3,8 +3,32 @@
...
 {
-    //
+    protected $foods;
+    
+    /**
+     * FoodController constructor.
+     *
+     * @param Food $foods dependence injection
+     */
+    public function __construct(Food $foods)
```
Comment (ghost) [inline app/Http/Controllers/FoodController.php]: -object should be a singular.

File: app/Http/Controllers/FoodController.php [hunk 8]
```diff
@@ -3,8 +3,32 @@
...
 
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+use App\Food;
+use App\Category;
 
 class FoodController extends Controller
 {
-    //
+    protected $foods;
```
Comment (ghost) [inline app/Http/Controllers/FoodController.php]: -object should be a singular.

File: resources/views/foods/index.blade.php [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+                                <th class="col-md-2">{{ trans('foods/listfoods.action') }}</th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                            @php ($i = 1)
+                            @foreach($foods as $food)
+                                <tr>
+                                    <td>{{ $i++ }}</td>
+                                    <td>{{ $food->name }}</td>
+                                    <td>{{ $food->categories->name }}</td>
```
Comment (ghost) [inline resources/views/foods/index.blade.php]: -you have to use with function to get category  when you get list food in controller.

**Review by ghost — CHANGES_REQUESTED** [review]
check comment.

**Comment by vhhieu95** [comment]
I updated, please check it again. Thank you so much!

File: composer.lock [hunk 10]
```diff
@@ -0,0 +1,3972 @@
+{
```
Comment (ghost) [inline composer.lock]: why do you commit this file?

File: resources/views/foods/index.blade.php [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+        <!-- /.box-header -->
+        <div class="box-body">
+            <div class="dataTables_wrapper form-inline dt-bootstrap">
+                <div class="row">
+                    <div class="col-sm-12">
+                        <table class="table table-bordered table-striped dataTable table-hover"
+                               role="grid">
+                            <thead>
+                            <tr role="row">
+                                <th class="col-md-1">{{ trans('foods/listfoods.id') }}</th>
```
Comment (ghost) [inline resources/views/foods/index.blade.php]: you should use ':' instead of '/'. please check it. thank you!

**Review by ghost — CHANGES_REQUESTED** [review]
Check comment.

**Review by ghost — APPROVED** [review]
check comment

File: app/Http/Controllers/FoodController.php [hunk 12]
```diff
@@ -3,8 +3,32 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+use App\Food;
+use App\Category;
```
Comment (ghost) [inline app/Http/Controllers/FoodController.php]: Do you use Category class in this file.

File: app/Http/Controllers/FoodController.php [hunk 13]
```diff
@@ -3,8 +3,32 @@
 namespace App\Http\Controllers;
 
 use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
```
Comment (ghost) [inline app/Http/Controllers/FoodController.php]: Do you use DB in this file?

**Review by ghost — CHANGES_REQUESTED** [review]
Sorry, I'm mistake. check comment. thank you.

File: app/Http/Controllers/FoodController.php (hunk 13 above)
Comment (vhhieu95) [reply app/Http/Controllers/FoodController.php]: I forget remove it. I update now. Thank you very much

**Review by ghost — APPROVED** [review]
LGTM

File: resources/views/foods/index.blade.php [hunk 14]
```diff
@@ -0,0 +1,61 @@
...
+                                <th class="col-md-2">{{ __('Name') }}</th>
+                                <th class="col-md-2">{{ __('Category') }}</th>
+                                <th class="col-md-1">{{ __('Price') }}</th>
+                                <th class="col-md-4">{{ __('Description') }}</th>
+                                <th class="col-md-2">{{ __('Action') }}</th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                            @php ($i = 1)
+                            @foreach($foods as $food)
```
Comment (at-duyho) [inline resources/views/foods/index.blade.php]: what do you display when foods is null?

**Review by ghost — CHANGES_REQUESTED** [review]
check comment

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**